### PR TITLE
Refactor pynamodb

### PR DESCRIPTION
**Pull request by tommymalmqvist**
Rewrite using only pynamodb. Added support for v1 api calls for backwards compability and also implemented v2 api-calls for future usage

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -1,66 +1,264 @@
...
-db_event = Dynamo.EventModel
-db_lock = Dynamo.LockModel
 
-# create tables
-for db in [db_event, db_lock]:
-    if not db.exists():
-        db.create_table(read_capacity_units=1, write_capacity_units=1, wait=True)
+for db_instance in [EventTable, LockTable]:
+    if not db_instance.exists():
+        db_instance.create_table(read_capacity_units=1, write_capacity_units=1, wait=True)
```
Comment (parberge) [inline app.py]: This capacity stuff might benefit from being possible to change (e.g load values from env variables if defined)

File: chalicelib/lib/db_v2.py [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    return json.dumps(events)
+
+
+def create_event_v2(events):
+    """
+    :param events:
+    :return: status
+    """
+    if isinstance(events, str):
+        events = ast.literal_eval(events)
```
Comment (parberge) [inline chalicelib/lib/db_v2.py]: Can you explain why `events` might be a string and what this `ast.literal_eval(events)` will do?

File: tests/test_api.py [hunk 3]
```diff
@@ -1,21 +1,225 @@
...
+    r = requests.post(f"{local_api}/events", json=event)
+    return r
+
+
+def create_lock(lock: dict):
+    r = requests.post(f"{local_api}/locks", json=lock)
+    return r
+
+
+def test_create_event():
```
Comment (parberge) [inline tests/test_api.py]: All these tests. Might it be valuable to check that the response returned is also valid json?

File: app.py (hunk 1 above)
Comment (tommymalmqvist) [reply app.py]: Good idea, should create a separate ticket for it

File: chalicelib/lib/db_v2.py (hunk 2 above)
Comment (tommymalmqvist) [reply chalicelib/lib/db_v2.py]: Hmm, dont know

https://github.com/codelabsab/timereport-api/commit/52be417fd263e2ab826d2a4be857915e4813ae6b

Kami introduced it some time ago

File: tests/test_api.py (hunk 3 above)
Comment (tommymalmqvist) [reply tests/test_api.py]: Yes, we should have tests on the data returned. Should have a ticket on that.

**Pull request merged**